app/javascript/components/timeline-options/timeline-helper.js

```diff
@@ -40,10 +40,16 @@ const smartAndOrStatements = (group, array) => {
 };
 
 /** Function to build the URL to get all events that fall under the user selected parameters. */
-export const buildUrl = (values) => {
+export const buildUrl = (values, pageFilters) => {
   // TODO: Different timeline show different data, ensure all necesary data is pulled
   let url = `/api/event_streams?limit=5000&offset=0&expand=resources`;
-  url += `&attributes=group,group_level,group_name,id,event_type,ems_id,type,timestamp,created_on,host,source`;
+  url += `&attributes=group,group_level,group_name,id,event_type,ems_id,type,timestamp,created_on,host,source,message`;
+  url += `,vm,ext_management_system`; // TODO heavy stuff to call in, but needed to be displayed in table
```

Comment:
I think we have `host_id` already, so I need to expose `name` as a virtual column so you don't have to check if it's there.

Same for ext_management_system and vm.  We can do that in a followup.